identity-service/src/relay/txRelay.js

```diff
@@ -273,6 +293,10 @@ const sendTransactionInternal = async (req, web3, txProps, reqBodySHA) => {
       Math.floor(Date.now() / 1000),
       JSON.stringify(redisLogParams)
     )
+    const end = new Date().getTime()
+    const totalTransactionLatency = end - startTransactionLatency
+    await reporter.reportError({ chain: 'poa', userId, contractAddress, nethermindContractAddress, senderAddress, totalTime: totalTransactionLatency, txSubmissionTime: relayStats.poa.txSubmissionTime, errMsg: e.toString() })
```

Comment:
I think we want to make these non-blocking here & above (don't await it)

otherwise, change looks great on my end!!